test/dialogs/dialog_form/helpers.js

```diff
@@ -77,7 +77,7 @@ function createRemoteDialog(id, server, config) {
   // Include any passed config items.
   if(config) {
     for(var prop in config) {
-      if(config.hasOwnProperty(prop)) {
+      if(config.hasownproperty(prop)) {
```

Comment:
Intentional change to lowercase that still works?

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/hasOwnProperty

(haven't read that page... just in case the answer to yes/no is in there)